FIPS/src/test/java/net/snowflake/client/TestUtil.java

```diff
@@ -47,7 +47,11 @@ public interface TestRunInterface {
    */
   public static String systemGetEnv(String env) {
     try {
-      return System.getenv(env);
+      String value = System.getenv(env);
+      if(value == null) {
+        value = System.getenv().getOrDefault(env, null);
+      }
+      return value;
```

Comment:
can we just reuse `SnowflakeUtil.systemGetEnv` in all the places where TestUtil.systemGetEnv is used?

Reply:
I will do that. Thank you

Reply:
I tried to remove the TestUlt.systemGetEnv, but it has issues when we try to validate the Old JDBC driver because that version is not available; that's why it was duplicated. https://github.com/snowflakedb/snowflake-jdbc/actions/runs/9352721781/job/25741532405?pr=1776

Reply:
Let's make the TestUtil method deprecated then with comment when we will be able to delete it

Reply:
@sfc-gh-dprzybysz Done